test/suites/auth.sh

```diff
@@ -143,6 +143,11 @@ EOF
   # Perform access check compatibility with project feature flags
   auth_project_features
 
+  # The OIDC identity should be able to delete themselves without any permissions.
+  lxc auth identity group remove oidc/test-user@example.com test-group
+  lxc_remote auth identity info oidc: | grep -Fq 'effective_permissions: []'
+  lxc_remote auth identity delete oidc:oidc/test-user@example.com
```

Comment:
We could also add some tests to check if the identity was indeed deleted as expected and (maybe) some tests to check that a user without permissions can't delete another. What do you think?

Reply:
Yep I'll double check that the identity is indeed deleted. As for checking that an identity without sufficient permissions cannot delete another, the difficulty here is that our `mini-oidc` only allows one user at a time. I think we can address this much more easily when the TLS identities are added. We can manage the identities in the testing scripts much more easily at that point.

Reply:
Ok, I see. Learning a lot from doing this review :))